virt/virt-4-11-release-notes.adoc

```diff
@@ -181,10 +181,26 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
-//New known issues for 4.11 go above this comment
+* Some persistent volume claim (PVC) annotations created by the Containerized Data Importer (CDI) can cause the virtual machine snapshot restore operation to hang indefinitely. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070366[*BZ#2070366*])
+** As a workaround, you can remove the annotations manually:
+. Obtain the xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virtual-machine-snapshot-controller-and-custom-resource-definitions-crds[VirtualMachineSnapshotContent] custom resource (CR) name from the `status.virtualMachineSnapshotContentName` value in the `VirtualMachineSnapshot` CR.
+. Edit the `VirtualMachineSnapshotContent` CR by removing all lines that contain `k8s.io/cloneRequest`.
```

Comment:
Suggestion for clarity (unsure if this is just a Me Thing):

```
. Edit the `VirtualMachineSnapshotContent` CR and remove all lines that contain `k8s.io/cloneRequest`.
```

(If this were a normal procedure, I'd also want to see steps for opening/viewing these CRs, but because it's a known issue, I think this is ok.)